content/en/docs/application-store/external-apps/deploy-metersphere.md

```diff
@@ -19,53 +19,53 @@ This tutorial demonstrates how to deploy MeterSphere on KubeSphere.
 
 ### Step 1: Add an app repository
 
-1. Log in to KubeSphere as `ws-admin`. In your workspace, go to **App Repos** under **Apps Management**, and then click **Add Repo**.
+1. Log in to KubeSphere as `ws-admin`. In your workspace, go to **App Repositories** under **App Management**, and then click **Add**.
 
-   ![add-repo](/images/docs/appstore/external-apps/deploy-metersphere/add-repo.PNG)
+   ![add-repo](/images/docs/appstore/external-apps/deploy-metersphere/add-repo.png)
```

Comment:
Double check all the images in the guide Deploy MeterSphere on KubeSphere. I reckon they are not accessible.